### DISCUSSION TITLE
Why the preview quality is too bad?

### DISCUSSION THREAD

**Discussion by exefile2** [TeamNewPipe/NewPipe#11807] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

The preview quality of a video while seeking is too bad and it's very difficult to see the preview during lectures. Is there any fixes? I have a strong wifi connection and don't know why this is happened...
![Screenshot_2024-12-20-21-04-31-900_org schabi newpipe](https://github.com/user-attachments/assets/eb3b8ee7-58c4-4e92-9e68-2b514f1069c3)

**Comment by asandikci** [comment]
Settings > Content > Image Quality
This is related to thumbnail but maybe affects the seekbar previews. Could you pls test

**Comment by ShareASmile** [comment]
Also check whether seekbar thumbnail preview is set to high quality.

Settings > Video and audio > seekbar thumbnail preview > change to high quality (larger) if not earlier set to high quality.

Edit: it looks like from screenshot it is already set to high quality(larger)

**Comment by exefile2** [comment]
It's already set to high quality (larger).

**Comment by ShareASmile** [comment]
What about
> Settings > Content > Image Quality
This is related to thumbnail but maybe affects the seekbar previews. Could you pls test
